litmus-portal/frontend/src/views/ChaosWorkflows/BrowseCluster/styles.ts

```diff
@@ -139,6 +140,98 @@ const useStyles = makeStyles((theme) => ({
   btnText: {
     paddingLeft: theme.spacing(1.625),
   },
+  // date picker
+  datePickerColor: {
+    color: theme.palette.secondary.dark,
+  },
+  // Table status
+  completed: {
+    width: '5.9125rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: theme.palette.primary.dark,
+    background: theme.palette.customColors.menuOption.active,
+  },
+  failed: {
+    width: '5.9125rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: theme.palette.error.dark,
+    backgroundColor: theme.palette.error.light,
```

Comment:
Here, css properties for both completed and active are same only background  color are differents.Try to use `${}${}` format.